package/MDAnalysis/coordinates/XYZ.py

```diff
@@ -97,41 +97,46 @@ class XYZWriter(base.Writer):
     # these are assumed!
     units = {'time': 'ps', 'length': 'Angstrom'}
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, filename, n_atoms=None, atoms='X', convert_units=None,
```

Comment:
At least according to the current [Trajectory API](http://www.mdanalysis.org/mdanalysis/documentation_pages/coordinates/init.html#trajectory-api), `**kwargs` needs to remain as a catchall. Readers [have to be able to ignore additional kwargs](http://www.mdanalysis.org/mdanalysis/documentation_pages/coordinates/init.html#id13).

As it is, this commit *breaks* the API.